pyqg/layered_model.py

```diff
@@ -113,15 +114,24 @@ def __init__(
         self.beta = beta
         self.rd = rd
         self.delta = delta
+
+        if U is None: U = (np.arange(nz) * 0.025)[::-1]
+        if V is None: V = np.zeros(nz)
+        if H is None: H = [500] + [1750 for _ in range(nz-1)]
+        if rho is None: rho = np.arange(nz) * 0.3 + 1025
```

Comment:
Why not just use default arguments in the constructor?

Reply:
Hmm -- that would be better in some respects, but using `None` and overwriting in the body of `__init__` allows `LayeredModel` to work no matter what `nz` is passed.

Reply:
gotcha that makes total sense.

Reply:
Awesome, will plan to merge then.